### PR TITLE
Press page: first section

### PR DESCRIPTION
**Pull request by natapg**
Closes https://github.com/fingerprintjs/fingerprint.com/issues/836
https://www.figma.com/file/Ad7wlJvqIMSOwWT5ung0Or/Website-New?node-id=206%3A4038&t=VQc0eGAJ3nrXQhsP-1

### CODE REVIEW

File: src/pages/resources/press/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    </LayoutTemplate>
+  )
+}
+
+export function Head(props: HeadProps) {
+  return (
+    <SEO
+      pathname={props.location.pathname}
+      title='Fingerprint Press & Brand Resources - Logos and Recent News'
+      description='Visit the Fingerprint Press and Brand Resources. This page includes recent news and media coverage, the company story, and downloaded brand guidelines and logos.'
```
Comment (tan-90) [inline src/pages/resources/press/index.tsx:30]: Copy:
```suggestion
      description='Visit the Fingerprint Press and Brand Resources. This page includes recent news and media coverage, the company story, and downloadable brand guidelines and logos.'
```

File: src/components/press/PressReleasesSection/PressReleasesSection.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  hrefUrl: string
+  box?: boolean
+}
+function Card({ title, company, hrefUrl, box }: CardProps) {
+  return (
+    <div className={classNames(styles.card, { [styles.box]: box })}>
+      <div>
+        <h6 className={styles.cardTitle}>{title}</h6>
+        <p className={styles.company}>{company}</p>
+      </div>
```
Comment (tan-90) [inline src/components/press/PressReleasesSection/PressReleasesSection.tsx:82]: I recently found out about `hgroup`, we can use it here.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup

**Review by tan-90 — APPROVED** [review]
Nice grids!
Just left 2 small notes, it looks good.

File: src/pages/resources/press/index.tsx (hunk 1 above)
Comment (natapg) [reply src/pages/resources/press/index.tsx:30]: @fpjsmarketing

File: src/components/press/PressReleasesSection/PressReleasesSection.tsx (hunk 2 above)
Comment (natapg) [reply src/components/press/PressReleasesSection/PressReleasesSection.tsx:82]: I didn't know that tag, I will use it a lot

**Pull request merged**